conftest.py

```diff
@@ -12,45 +10,40 @@
 
 
 @pytest.fixture(scope='function')
-def mozwebqa_logged_in(request):
+def login(request, mozwebqa, variables):
+    user = variables['users']['default']
     from pages.login_page import MozTrapLoginPage
-    mozwebqa = request.getfuncargvalue('mozwebqa')
     login_pg = MozTrapLoginPage(mozwebqa)
     login_pg.go_to_login_page()
-    login_pg.login()
+    login_pg.login(user['email'], user['password'])
+
 
-    return mozwebqa
+@pytest.fixture
+def api(request, variables):
+    url = request.getfuncargvalue('mozwebqa').base_url
+    return MoztrapAPI(variables['api']['user'], variables['api']['key'], url)
 
 
 @pytest.fixture(scope='function')
-def product(request):
-    """Return a product created via the Moztrap API, and automatically delete the product after the test."""
-    mozwebqa = request.getfuncargvalue('mozwebqa')
-    credentials = mozwebqa.credentials['default']
-    request.product = MockProduct()
-    api = MoztrapAPI(credentials['api_user'], credentials['api_key'], mozwebqa.base_url)
-    api.create_product(request.product)
-
-    # This acts like a tearDown, running after each test function
+def product(request, api):
+    """Return a product created via the API."""
+    product = MockProduct()
```

Comment:
I'm not entirely sure where/how it's stored, but we don't need to store it ourselves. I made sure to test this too.